src/components/ShareStructure/EditShareStructure.vue

```diff
@@ -0,0 +1,490 @@
+<template>
+  <div>
+
+    <confirm-dialog
+      ref="confirm"
+      attach="#app"
+    />
+
+    <v-expand-transition id="addShareStructureContainer">
```

Comment:
It might be nice to:
- move this id to the upper div
- name it after the file name
- attach the Confirm Dialog to it (I don't know if `#app` is visible in this file)

Reply:
Yes, the attach was an error, good catch!
Fixed in next commit